integrations/github/review_comments.go

```diff
@@ -0,0 +1,270 @@
+package github
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/v60/github"
+
+	"go.autokitteh.dev/autokitteh/sdk/sdkmodule"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+// https://docs.github.com/en/rest/pulls/comments#create-a-review-comment-for-a-pull-request
+func (i integration) createReviewComment(ctx context.Context, args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error) {
+	// Parse the input arguments.
+	var (
+		owner, repo          string
+		pullNumber           int
+		body, commitID, path *string
+
+		position, line, startLine, inReplyTo *int
+		side, startSide, subjectType         *string
+	)
+
+	err := sdkmodule.UnpackArgs(args, kwargs,
+		"owner", &owner,
```

Comment:
so many. we should consider just getting a dict or a struct here someday.

Reply:
Done - using the request struct directly during unpacking